students/zelichowski_michal/lesson_02_flow_control/the_number_of_elements_equal_to_the_maximum.py

```diff
@@ -0,0 +1,15 @@
+"""A sequence consists of integer numbers and ends with the number 0.
+Determine how many elements of this sequence are equal to its largest
+element. """
+
+a = int(input())
+max = 0
+numberOfMax = 0
```

Comment:
Zerknij sobie na wytyczne PEP8 odnośnie naming convention dla zmiennych: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names
Zalecam poprawkę na `number_of_max` zgodnie z PEP8